tests/test_crs.py

```diff
@@ -518,6 +518,13 @@ def test_linear_units_factor():
         CRS.from_epsg(4326).linear_units_factor
 
 
+def test_units_factor():
+    assert CRS.from_epsg(3857).units_factor == ("metre", 1.0)
+    assert CRS.from_epsg(4326).units_factor == (
+        "degree", pytest.approx(0.017453292519943295)
+    )
```

Comment:
Before @sgillies says it I will! could you parametrize this tests (see https://github.com/developmentseed/morecantile/blob/master/tests/test_utils.py#L11-L22) 🙏

Reply:
That is a good catch :+1:. I will plan on updating it when I get a moment.